### ISSUE TITLE
[Functional bug] CRTL-U failed to open accessibility menu

### ISSUE THREAD

**Issue by logoshot** [topcoder-platform/Training-Challenges#142]
**Steps to Reproduce:**
- Go to https://www.topcoder.com/
- press CRTL+U

**Actual Result:**
Nothing happened

**Expected Result:**
Open the accessibility menu as it is shown in the accessibility menu
![image](https://user-images.githubusercontent.com/34298372/83343303-0f124c00-a32b-11ea-809b-3d17de4ef2a8.png)

**Device/OS/Browser Information:**
laptop/win10/chrome

**Comment by nithyaasworld** [comment]
Good Job @qunidad 

But this has already been reported earlier. https://github.com/topcoder-platform/Training-Challenges/issues/96

Please go ahead and check the following pages to know about the current and upcoming QA Challenges and participate on them for Big prize money and Cool QA Experience.

https://www.topcoder.com/challenges
https://www.topcoder.com/community/pipeline

**Comment by nithyaasworld** [comment]
I would also suggest you to explore all the **functions, pages, UI** that are in scope and not just focus on one-part of the app.

If you think there is no new bug, try in different browsers like FireFox, Safari of Mac, Edge, mobile browsers. Try negative scenarios like tapping on any button more than once, or by entering invalid inputs to the forms. Try editing and saving some data to your profile and then log off and re-login and verify if the data is still preserved. Try the search bars and filters wherever its present. Apply different combination of filters and check. Check if all the links in all the pages are working and not broken. Check if the pages are loading at a considerable time. 

Have a quick glance at the list of existing bugs and see if there are any part of the app that is still not tested. 

Keep trying different ways until you find a bug! Good Luck @qunidad

**Comment by qunidad** [comment]
think you for your advice @nithyaasworld
